### ISSUE TITLE
Defer to Bitcoin Core's default ports?

### ISSUE THREAD

**Issue by Sjors** [fort-nix/nix-bitcoin#667]
```
services.bitcoind {
    enable = true;
    regtest = true;
}
```

This should listen on port 18444, not 8332.

The RPC should be on port 18443 not 8332.

It might be better to not hardcode ports in the config file. I doubt the defaults will ever change. You could also set the correct values.

One benefit of using the default ports is that, if you want to e.g. run both a mainnet and testnet node, commands like `bitcoin-cli -test getblockchaininfo` will work.

**Comment by jonasnick** [comment]
Yes, the bitcoind module should use the normal regtest ports. It's easier for us though to set them explicitly, because we need to access the bitcoind module port options from other modules. See #671.

**Comment by erikarvstedt** [comment]
Fixed in #671.
